com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveClasspathsHandler.java

```diff
@@ -153,10 +156,18 @@ public void acceptSearchMatch(SearchMatch match) {
      * @throws CoreException
      *             CoreException
      */
-    private static String[] computeClassPath(IJavaProject javaProject) throws CoreException {
+    private static String[][] computeClassPath(IJavaProject javaProject) throws CoreException {
         if (javaProject == null) {
             throw new IllegalArgumentException("javaProject is null");
         }
-        return JavaRuntime.computeDefaultRuntimeClassPath(javaProject);
+        String[][] result = new String[2][];
```

Comment:
Suggestion only: Could use org.apache.commons.lang3.tuple/pair  for better reading.

Reply:
In typescript, tuple is like `[10, 11]`, but in org.apache.commons.lang3.tuple.Pair, tuple is like `(10, 11)`, the serialization/deserialization is not compatile, i would prefer to keep current implementation.